index.html

```diff
@@ -132,22 +145,23 @@ <h4>Tonic</h4>
                 <div class="work-description">
                     <h4>Multi-Post Stories</h4>
                     <ul class="description-element">
-                        <li>CANOPY</li>
+                        <li>FACEBOOK</li>
                         <li class="element">
                             <img class="bullet" src="./images/bullet.png" alt="bullet" />
                         </li>
-                        <li class="element">Back End Dev</li>
+                        <li class="element">Full Stack Dev</li>
                         <li class="element">
                             <img class="bullet" src="./images/bullet.png" alt="bullet" />
                         </li>
                         <li class="element">2015</li>
                     </ul>
```

Comment:
Kindly make sure that your design remain the same when resizing screen the list of item in this section cards are overlapping  set an horizontal margin   to these list items 

![card list items](https://user-images.githubusercontent.com/59707859/221054713-2495a0ae-7d44-4ced-8ddc-5091fe50e1db.PNG)